### PR TITLE
[DOCFIX] Fix "building from source" doc

### PR DESCRIPTION
**Pull request by jenoudet**
see title

### CODE REVIEW

File: docs/en/contributor/Building-Alluxio-From-Source.md [hunk 1]
```diff
@@ -94,12 +95,11 @@ Once Alluxio is built, you can validate and start it with:
 ```console
 $ # Alluxio uses ./underFSStorage for under file system storage by default
 $ mkdir ./underFSStorage
-$ ./bin/alluxio validateEnv local
```
Comment (LuQQiu) [inline docs/en/contributor/Building-Alluxio-From-Source.md:97]: why this is removed?

**Review by LuQQiu — APPROVED** [review]
LGTM, thanks for improving this doc

File: docs/en/contributor/Building-Alluxio-From-Source.md (hunk 1 above)
Comment (jenoudet) [reply docs/en/contributor/Building-Alluxio-From-Source.md:97]: When cloning the repo it produces `1 failures 4 warnings 7 skipped` as output, but the failure does not actually impede the process of building it from source.

**Comment by jenoudet** [comment]
@Xenorith I think this is ready for merge and cherry-pick

**Comment by Xenorith** [comment]
alluxio-bot, merge this please

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.7 please

**Pull request merged**